src/Data/Rational/Properties.agda

```diff
@@ -686,6 +718,65 @@ private
   { isAbelianGroup = +-0-isAbelianGroup
   }
 
+------------------------------------------------------------------------
+-- Properties of _+_ and -_
+
+neg-distrib-+ : ∀ p q → - (p + q) ≡ (- p) + (- q)
+neg-distrib-+ p q = toℚᵘ-injective (begin-equality
+  toℚᵘ(- (p + q))                    ≈⟨ toℚᵘ-homo‿- (p + q) ⟩
+  ℚᵘ.- toℚᵘ(p + q)                   ≈⟨ ℚᵘ.-‿cong (toℚᵘ-homo-+ p q) ⟩
+  ℚᵘ.- (toℚᵘ(p) ℚᵘ.+ toℚᵘ(q))        ≡⟨ ℚᵘ.neg-distrib-+ (toℚᵘ p) (toℚᵘ q) ⟩
+  (ℚᵘ.- toℚᵘ(p)) ℚᵘ.+ (ℚᵘ.- toℚᵘ(q)) ≈⟨ ℚᵘ.≃-sym (ℚᵘ.+-cong (toℚᵘ-homo‿- p) (toℚᵘ-homo‿- q)) ⟩
+  toℚᵘ(- p) ℚᵘ.+ toℚᵘ(- q)           ≈⟨ ℚᵘ.≃-sym (toℚᵘ-homo-+ (- p) (- q)) ⟩
+  toℚᵘ((- p) + (- q))             ∎
+  )
+  where
+  open ℚᵘ.≤-Reasoning
```

Comment:
According to the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md), the last four lines should be condensed to two lines as follows:
```
  toℚᵘ((- p) + (- q))                ∎)
  where open ℚᵘ.≤-Reasoning
```
and the `∎` should be aligned to the equational reasoning. There are other places in this PR where this applies as well!